app/config.js

```diff
@@ -1,17 +1,15 @@
 var coins = require("./coins.js");
 
-var currentCoin = "BTC";
+var currentCoin = process.env.BTCEXP_COIN || "BTC";
 
 var credentials = require("./defaultCredentials.js");
-var overwriteCredentials = require("./credentials.js");
-
-for (var key in overwriteCredentials) {
-	credentials[key] = overwriteCredentials[key];
-}
+try {
+  Object.assign(credentials, require("./credentials.js"))
+} catch (err) {}
 
 module.exports = {
-	cookiePassword: "0x000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f",
-	demoSite: true,
+	cookiePassword: process.env.BTCEXP_COOKIE_PASSWORD || "0x000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f",
+	demoSite: !process.env.BTCEXP_NO_DEMO,
```

Comment:
Wouldn't it be better to call the param `BTCEXP_DEMO`. Multiple negations can be confusing, it's better to make it more direct IMO.

Reply:
I initially did it like that to keep the default behavior the same as it is today (enable `demoSite` if nothing is explicitly set), but I ended up changing it in a [later commit](https://github.com/janoside/btc-rpc-explorer/pull/89/commits/f51ce159d264130ff2d9183bf3be1e47e1b18922) (part of #89). Should I backport this change there?

Reply:
@sangaman Yea, let's go back to what you had before, to avoid the double-negative as @diegogurpegui mentioned. By the way, thanks to you both for pushing this forward.